app.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "MusicStore",
+    "description": "ASP.NET 5 sample app",
+    "logo": "https://raw.githubusercontent.com/aspnet/MusicStore/master/src/MusicStore/wwwroot/Images/home-showcase.png",
```

Comment:
It's gone!

On Wed, Mar 4, 2015 at 10:05 AM, Praburaj Thiagarajan <
notifications@github.com> wrote:

> In app.json
> <https://github.com/aspnet/MusicStore/pull/411#discussion_r25797070>:
>
> > @@ -0,0 +1,8 @@
> > +{
> > +    "name": "MusicStore",
> > +    "description": "ASP.NET 5 sample app",
> > +    "logo": "https://raw.githubusercontent.com/aspnet/MusicStore/master/src/MusicStore/wwwroot/Images/home-showcase.png",
>
> Yeah. It will be good to remove it for maintainability [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aspnet/MusicStore/pull/411/files#r25797070>.
>



-- 
http://friism.com/
(650) 318-1051
Sapere aude